codegen/winterfell/src/air/mod.rs

```diff
@@ -13,8 +13,8 @@ use periodic_columns::add_fn_get_periodic_column_values;
 mod boundary_constraints;
 use boundary_constraints::{add_fn_get_assertions, add_fn_get_aux_assertions};
 
-mod transition_constraints;
-use transition_constraints::{add_fn_evaluate_aux_transition, add_fn_evaluate_transition};
+mod integrity_constraints;
+use integrity_constraints::{add_fn_evaluate_aux_transition, add_fn_evaluate_transition};
```

Comment:
I think it might be better not to rename these, since Winterfell doesn't have an idea of `integrity constraints`, it only has `boundary` or `transition` constraints. I think that our codegen crates will probably have the most clarity if they generally use the same language as the target. This is just my opinion though. What do you think?

If we agree on this, we should restore the comments below and the text in the README as well

Reply:
Makes sense. Will revert. Thanks.